src/Sylius/Bundle/CoreBundle/spec/Handler/CartCurrencyChangeHandlerSpec.php

```diff
@@ -58,21 +61,21 @@ function it_throws_handle_exception_when_a_cart_is_not_found(CartContextInterfac
 
     function it_handles_cart_currency_code_change(
         CartContextInterface $cartContext,
-        OrderUpdaterInterface $orderUpdater,
-        EventDispatcherInterface $eventDispatcher,
+        OrderUpdaterInterface $exchangeRateUpdater,
```

Comment:
`$exchangeRateUpdater` -> `$orderExchangeRateUpdater` - it does not update the exchange rate, but only updates the order's exchange rate.